INFO     root:Log.py:79 [INFO 2019-11-03 20:22:57]初始化关联文件
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:22:57]读取初始关联文件内容：   ItemsView(<Section: test_data>)
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:22:57]初始关联内容数据处理后：   {'name': '7', 'type': 'Web', 'version': 'KpmAX7lPs0', 'description': 'api_auto_8EP4hV95nw', 'project_id': '623116cf4d274453ba803e753fda2e69', 'token_id': 'd94f0bd8cc2b4ab1a76b23db16cdab1c', 'project_name': 'LJSMhC0wV8enTEzxcWoj', 'network_name': 'k1QBU5ZoHq9PMKm8pOaW_netwrok', 'subnet_name': 'SJtzZcALm8l1doHE3wrO_subnet', 'network_id': 'af23f0b5-978f-4f65-9c8c-ed7685b92bc4', 'subnet_id': 'd4d29468-e3f8-4471-83d0-c908424ebb4f', 'image_id': 'b4e9e907-686e-440d-bc1a-d73a8decfa54', 'flavor_id': 'flavor-1-1-1', 'server_name': 'wuzs_zG7eEqUM8WbOD24PIFTx', 'server_id': '952e41e1-bf51-4648-af18-d59ca5d83557', 'project_token_name': 'KPYIfUOQAJdSBT7s41cM', 'admin_id': '0ec82b7380cb4d0c80058501938cdf48', 'admin_role_id': 'cc33f2c3a47247329d0ee7043c4f475f', 'volume_name': 'volume_n684o15cZftkQreNOq9x', 'volume_id': '966dbf91-6235-4dcc-a30f-05a956b7a173', 'server_snap_name': 'snap_8JkrX4yjSduLBPHcmTxC', 'server_snap_id': '', 'volume_snapshot_id': '2cd6c8e7-0a47-4d18-b4b7-e445790b6b6f', 'network_name_for_check': '5yP1bXxrJVpowg8G2EsT_netwrok'}
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]====================================================================================================
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:22:59]请求头处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': 'd94f0bd8cc2b4ab1a76b23db16cdab1c'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:22:59]请求参数处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': 'd94f0bd8cc2b4ab1a76b23db16cdab1c'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:22:59]host处理结果：  192.168.54.120
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]请求接口：删除子网后查看子网
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]请求地址：http://192.168.54.120:9696/v2.0/subnets/d4d29468-e3f8-4471-83d0-c908424ebb4f
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]请求头: {'Content-Type': 'application/json', 'X-Auth-Token': 'd94f0bd8cc2b4ab1a76b23db16cdab1c'}
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]请求参数: {'subnet': {'admin_state_up': True}}
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]GET请求接口
DEBUG    urllib3.connectionpool:connectionpool.py:205 Starting new HTTP connection (1): 192.168.54.120:9696
DEBUG    urllib3.connectionpool:connectionpool.py:393 http://192.168.54.120:9696 "GET /v2.0/subnets/d4d29468-e3f8-4471-83d0-c908424ebb4f?subnet=admin_state_up HTTP/1.1" 404 135
INFO     root:Log.py:79 [INFO 2019-11-03 20:22:59]接口请求结果：
                         (404, {'NeutronError': {'message': 'Subnet d4d29468-e3f8-4471-83d0-c908424ebb4f could not be found', 'type': 'SubnetNotFound', 'detail': ''}})